### PR TITLE
chore: fix typos in documentation and test output

### PR DESCRIPTION
**Pull request by sohanian**
Fixes some typos in documentation and test output.

### CODE REVIEW

File: packages/fast-web-utilities/README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ The `isNullOrWhiteSpace` function determines if the specified string is undefine
 ```js
 import { isNullOrWhiteSpace } from "@microsoft/fast-web-utilities";
 
-const myAnchor = document.getElementById("#id");
+const myAnchor = document.querySelector("#id");
```
Comment (chrisdholt) [inline packages/fast-web-utilities/README.md]: Was this just preference? Doesn't seem related to the overall PR

**Comment by nicholasrice** [comment]
@sohanian thanks for the PR! Once you rebase against current master we can get this change in.

**Comment by sohanian** [comment]
I had to figure out the "rebase from master." I think I got it now.

**Comment by sohanian** [comment]
Alright, I tried again and hope it's ok. I'll try discord tomorrow if not, too late on east coast for me now.

**Comment by chrisdholt** [comment]
Thanks @sohanian for contributing!
